test/e2e/quick_start_test.go

```diff
@@ -99,3 +99,17 @@ var _ = Describe("When following the Cluster API quick-start with Ignition", fun
 		}
 	})
 })
+
+// NOTE: This test requires a dualstack management cluster (can be configured via IP_FAMILY=dual).
+var _ = Describe("When following the Cluster API quick-start with dualstack [dualstack] [PR-Informing]", func() {
```

Comment:
Are we using the dualstack tag somewhere, if we don't I would prefer not defining it

We currently only have a very limited set of tags, which are the ones we actually need to configure the Prow jobs.

I would prefer not expanding this as long as we don't have a clear idea / concept behind tags.